src/OpenAL/Silk.NET.OpenAL/AL/AL.cs

```diff
@@ -350,11 +349,33 @@ public static AL GetApi(bool soft = false)
             return ret;
         }
 
+        /// <summary>
+        ///     Attempts to load a native OpenAL extension of type <typeparamref name="T" />.
+        /// </summary>
+        /// <param name="ext">
+        ///     The loaded extension.
+        /// </param>
+        /// <typeparam name="T">
+        ///     Type of <see cref="NativeExtension{T}" /> to load.
+        /// </typeparam>
+        /// <returns>
+        ///     <c>True</c> if the extension was loaded, otherwise <c>False</c>.
+        /// </returns>
```

Comment:
style:
- `summary` is always in "paragraph format" where the tags are on separate lines to the content
- `param`s and `typeparam`s are on the same line as their tags except where that would exceed 120 characters on that line, in which case "paragraph format" is used.
- "paragraph format" shouldn't have an indent (unlike regular XML)
- `c` and `code` elements should be valid C# code.

```suggestion
        /// <summary>
        /// Attempts to load a native OpenAL extension of type <typeparamref name="T" />.
        /// </summary>
        /// <param name="ext">The loaded extension.</param>
        /// <typeparam name="T">Type of <see cref="NativeExtension{T}" /> to load.</typeparam>
        /// <returns><c>true</c> if the extension was loaded, otherwise <c>false</c>.</returns>
```